gsk4/src/render_node.rs

```diff
@@ -0,0 +1,161 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use crate::RenderNodeType;
+use glib::translate::*;
+use glib::StaticType;
+use std::fmt;
+use std::ptr;
+
+glib::wrapper! {
+    #[derive(Debug, PartialEq, Eq, PartialOrd, Ord, Hash)]
+    pub struct RenderNode(Shared<ffi::GskRenderNode>);
+
+    match fn {
+        ref => |ptr| ffi::gsk_render_node_ref(ptr),
+        unref => |ptr| ffi::gsk_render_node_unref(ptr),
+        get_type => || ffi::gsk_render_node_type_get_type(),
+    }
+}
+
+impl RenderNode {
+    //#[doc(alias = "gsk_render_node_deserialize")]
+    //pub fn deserialize(bytes: &glib::Bytes, error_func: /*Unimplemented*/FnMut(/*Ignored*/ParseLocation, /*Ignored*/ParseLocation, &glib::Error), user_data: /*Unimplemented*/Option<Fundamental: Pointer>) -> Option<RenderNode> {
+    //    unsafe { TODO: call ffi:gsk_render_node_deserialize() }
```

Comment:
This looks relatively easy to add, maybe create an issue if you don't want to do it as part of this PR.

Reply:
It's handled by #157 already